infrastructure/helm/quantumserverless/charts/manager/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
I think we do not even need it. 
It's better to do expose ping endpoint in manager and add healthcheck in helm chart. We can do it later